### PR TITLE
[create] feat: allow image tag override

### PR DESCRIPTION
**Pull request by naseemkullah**
While using the chart's appVersion as image tag is the sanest default, it is not uncommon to want to override this if using a custom image, or using helm to manage  an in-house app running different tags across different environments.

cc @scottrigby @unguiculus @rimusz @mattfarina etc.

### CODE REVIEW

**Comment by rimusz** [comment]
thanks @naseemkullah it looks as a good practices example to have in the `create` of the chart

**Review by hickeyma — APPROVED** [review]
LGTM thanks @naseemkullah. I would like to get some feedback from @helm/charts-maintainers before merging.

**Review by unguiculus — APPROVED** [review]
I like it. I was wondering anyways why `image.tag` had been dropped. Using `.Chart.AppVersion` makes only sense for using public charts when it should always match `image.tag`. For inhouse charts I always remove `AppVersion` completely because you pass in `image.tag` during deployment.

So, yes, let's merge this. 🎉

**Pull request merged**
